TurkishFilter/sections/general_extensions.txt

```diff
@@ -302,6 +302,9 @@ mygaziantep.com#$#body.modal-open { overflow: visible!important; padding-right:
 kayseristarhaber.com.tr#$##modal-ads { display: none !important; }
 kayseristarhaber.com.tr#$#.modal-backdrop { display: none !important; }
 kayseristarhaber.com.tr#$#body { overflow: visible !important; }
+! https://github.com/AdguardTeam/AdguardFilters/issues/63717
+tercihrobotu.com.tr#$#.Banner { border: none !important; }
+tercihrobotu.com.tr#$#a[href$="#ReklamRezervasyon"] { opacity:0 !important; }
```

Comment:
It seems that this element is still clickable.
What about changing `opacity: 0` to `visibility: hidden`, so this ad will be not clickable?